subsys/bluetooth/host/l2cap.c

```diff
@@ -82,8 +82,14 @@ struct bt_l2cap {
 	struct bt_l2cap_le_chan	chan;
 };
 
+static struct bt_l2cap_cb l2cap_callbacks;
 static struct bt_l2cap bt_l2cap_pool[CONFIG_BT_MAX_CONN];
 
+void register_l2cap_callbacks(struct bt_l2cap_cb cb)
```

Comment:
typically we pass const pointer to structure with callbacks instead of copy like here